std_msgs/Header header
string[] name
string[] state
string[] error
float32[] position
float32[] velocity
float32[] current
uint8[] limit_hit"